### PR TITLE
fix: Okex parseFuturesAccount for free balance

### PR DESCRIPTION
**Pull request by imdgr886**
Incorrect calculation for futures's free balance.  
Docs: https://www.okex.com/docs/en/#futures-singleness-single

### CODE REVIEW

**Comment by kroitor** [comment]
@imdgr886 thx! As you've commented, it may be a bad idea to use `free`, `used`, `total` for contract trading and margin accounts. What unified structure would you suggest for the contract/margin unification schema?

**Comment by imdgr886** [comment]
I just want to get the available balance of the entire account
image

**Comment by imdgr886** [comment]
It not equal to `total_avail_balance`

File: js/okex.js [hunk 1]
```diff
@@ -1593,9 +1593,24 @@ module.exports = class okex extends Exchange {
             const code = this.safeCurrencyCode (id);
             const balance = this.safeValue (info, id, {});
             const account = this.account ();
+            const totalAvailBalance = this.safeFloat (balance, 'total_avail_balance');
+            if (this.safeString(balance, 'margin_mode') === 'fixed') {
+                const contracts = this.safeValue(balance, 'contracts', []);
+                let free = totalAvailBalance;
+                for (let i = 0; i < contracts.length; i++) {
+                    let cont = contracts[i];
+                    free += this.safeFloat (cont, 'fixed_balance') + this.safeFloat (cont, 'realized_pnl')
```
Comment (kroitor) [inline js/okex.js:1602]: We have to implement the `+` operations with `this.sum`, as explained here:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Comment (imdgr886) [reply js/okex.js:1602]: Im sorry, I can't pull the reposition to my pc, so can't run eslint to check.

Comment (kroitor) [reply js/okex.js:1602]: I'd suggest to split line 1602 into basic statements:

```JavaScript
free += this.safeFloat (cont, 'fixed_balance') + this.safeFloat (cont, 'realized_pnl') - this.safeFloat (cont, 'margin_frozen') - this.safeFloat (cont, 'margin_for_unfilled');
```

↓

```JavaScript
const fixedBalance = this.safeFloat (cont, 'fixed_balance');
const realizedPnl = this.safeFloat (cont, 'realized_pnl');
const marginFrozen = this.safeFloat (cont, 'margin_frozen');
const marginForUnfilled = this.safeFloat (cont, 'margin_for_unfilled');
const margin = this.sum (fixedBalance, realizedPnl) - marginFrozen - marginForUnfilled;
free = this.sum (free, margin);
```

**Pull request merged**